sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -81,18 +81,31 @@ class PlanParserSuite extends AnalysisTest {
     assertEqual("select * from a intersect all select * from b", a.intersect(b, isAll = true))
   }
 
+  private def cte(plan: LogicalPlan, namedPlans: (String, (LogicalPlan, Seq[String]))*): With = {
+    val ctes = namedPlans.map {
+      case (name, (cte, columnAliases)) =>
+        val subquery = if (columnAliases.isEmpty) {
+          cte
+        } else {
+          UnresolvedSubqueryColumnAliases(columnAliases, cte)
+        }
+        name -> SubqueryAlias(name, subquery)
+    }
+    With(plan, ctes)
+  }
```

Comment:
This test suite is updated a few minute ago. Could you rebase once more?
There will be another `cte` function in this test suite.

Reply:
Oh, this PR is already updated to the master. Then, please move this function to the below of the existing `cte` function. We had better gather the similar functions. If possible, please consolidate them into one.

Reply:
Fixed, thanks.